app/Http/Controllers/AuthController.php

```diff
@@ -5,24 +5,14 @@
 use App\Http\Controllers\Controller;
 use App\Http\Requests\RegisterRequest;
 use App\Models\User;
-use Illuminate\Support\Facades\Validator;
 
 
 
 class AuthController extends Controller
 {
     
    public function register(RegisterRequest $request)  { 
-    $validator= Validator::make($request->all(),[
-        'email' => 'required|email',
-        'password' => 'required|min:6|max:100',
-        'confirm_password' => 'required|same:password'
-    ]);
-    if ($validator->fails()) {
-        return response() ->json([
-            'errors' =>$validator->errors()
-        ],422);
-    }
+
     $user = User::create([
```

Comment:
$user = User::create($request->validated()); 
$request->validated() returns an array of validated values.

you must hash the user password before saving. investigate this one:
https://gist.github.com/mhanson01/8115466